manager/integration/tests/test_basic.py

```diff
@@ -13,6 +15,47 @@
 from common import get_volume_endpoint, get_volume_engine
 from common import get_random_client
 from common import CONDITION_STATUS_FALSE, CONDITION_STATUS_TRUE
+from common import RETRY_COMMAND_COUNT
+
+
+def create_volume(client, vol_name, num_of_replicas=2,
```

Comment:
Can this one be merged with `client.create_volume`?

Reply:
no. client is imported from `cattle`